scripts/trips_requested_lookup_from_response.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
```

Comment:
The plan was to create something similar to this but which loaded multiple of the JSON responses files and created the out metrics / matrix from those as the JSON files should be updated as each response is recieved. Most of the functionality for
calculating cost metrics / matrix from the responses file already exists in `cost.cost_matrix_from_responses` function it would just need updating to handle multiple files.

Reply:
Updated by [2bcb0cb](https://github.com/Transport-for-the-North/otp4gb-py/commit/2bcb0cbeab8c5102221a03f1a7c77665e5435e4e)

This has been changed a bit into two scripts now. 

[cost_metrics_from_responses.py](https://github.com/Transport-for-the-North/otp4gb-py/blob/OTP_v2_TRSE_/scripts/cost_metrics_from_responses.py) which replaces export_costs_from_partial_run.py. This script compiles a single cost-metrics matrix from a list of OTP responses files (complete or not) now making use of imported cost module. 

[trips_requested_from_cost_metrics.py](https://github.com/Transport-for-the-North/otp4gb-py/blob/OTP_v2_TRSE_/scripts/trips_requested_from_cost_metrics.py) which compiles a single cost-metrics matrix from a list of cost-metrics paths (useful if numerous OTP runs have completed successfully and don't want to have to wait for the cost-metrics matrices to be re-compiled from multiple responses files using script above).

Most importantly this script also outputs a list of unique trips from the compiled cost-metrics matrix, which can be used as a lookup to prevent the re-requesting of these trips in a later run.